Remove app deployment guides from developer docs (#1307)
mattermost/mattermost-developer-documentation: site/content/integrate/apps/authentication/mattermost-to-app/_index.md

weight: 20
---

> - HTTP apps can request that Mattermost includes a secret-based JWT in each request. See [HTTP Deployment]("https://docs.mattermost.com/guides/deploy-http.html") for more.

Reviewer: ```suggestion
- HTTP apps can request that Mattermost includes a secret-based JWT in each request. See {{< newtabref href="https://docs.mattermost.com/guides/deploy-http.html" title="HTTP Deployment" >}} for details.
```

> - HTTP apps can request that Mattermost includes a secret-based JWT in each request. See {{< newtabref href="https://docs.mattermost.com/guides/deploy-http.html" title="HTTP Deployment" >}} for details.

Reviewer: ```suggestion
- HTTP apps can request that Mattermost includes a secret-based JWT in each request. See the {{< newtabref href="https://docs.mattermost.com/guides/deploy-http.html" title="HTTP Deployment" >}} documentation for details.
```

> - AWS apps are invoked via the AWS Lambda `Invoke` and S3 `Get` APIs, which are authenticated with "invoke" AWS credentials (access/secret keys). See [AWS Deployment]("https://docs.mattermost.com/guides/deploy-aws.html") for more.

Reviewer 2: @cwarnermm I think I understand why this is here. We are linking two different systems here - the developer site and the docs site. This site builder for the dev docs isn't aware of "shortcodes" for docs website AFAIK. So we have to hardcode what's currently in production. Maybe we can modularize the code somehow to generate the link a different way?

> - AWS apps are invoked via the AWS Lambda `Invoke` and S3 `Get` APIs, which are authenticated with "invoke" AWS credentials (access/secret keys). See [AWS Deployment]("https://docs.mattermost.com/guides/deploy-aws.html") for more.

Reviewer: AHA! I believe you're correct, @mickmister, thank you! Yes -- for every link that would have taken users to another page of the dev docs, we need to take them to docs.mm.com instead.

In the generated preview, these quoted links don't resolve as expected. Not sure yet if that's due to the quotation marks for these links, or due to a limitation of the generated docs build.

I'll review more deeply on Monday to find a suitable way forward. @codeEmpress1 - No action is required of you :) Thank you again for your contributions!!

> - AWS apps are invoked via the AWS Lambda `Invoke` and S3 `Get` APIs, which are authenticated with "invoke" AWS credentials (access/secret keys). See [AWS Deployment]("https://docs.mattermost.com/guides/deploy-aws.html") for more.

Reviewer: ```suggestion
- AWS apps are invoked via the AWS Lambda `Invoke` and S3 `Get` APIs, which are authenticated with "invoke" AWS credentials (access/secret keys). See the {{< newtabref href="https://docs.mattermost.com/guides/deploy-aws.html" title="AWS Deployment" >}} documentation for details.

[AWS Deployment]("https://docs.mattermost.com/guides/deploy-aws.html") for more.
```
